sig/tools/constitution-zh_CN.md

```diff
@@ -34,7 +34,8 @@ Ecosystem SIG 聚焦 TiKV 项目的周边模块，例如 [client-rust](https://g
   - 每 2 周以文档形式同步一次当前各个项目的开发进度。
   - 每 2 周召开一次全组进度会议，时间依据参会人员可用时间另行协商。目前没有项目正在开发的成员可选择性参加以便了解各个项目进度。若参与开发的成员不能参加，需提前请假且提前将自己的月度进度更新至文档。
   - 每次会议由一名成员进行会议记录，在会议结束 24 小时内完成会议记录并公开。会议记录由小组成员轮流执行。
```

Comment:
tools team currently do not hold these regular public meetings yet. had the tikv ecosystem team held such meetings before?

Reply:
No, thanks for pointing it out!